hypothesis-python/tests/codemods/test_codemods.py

```diff
@@ -75,6 +75,14 @@ def test_substitution(self) -> None:
         """
         self.assertCodemod(before=before, after=after)
 
+    def test_noop_with_new_floats_kw(self) -> None:
+        before = """
+            import hypothesis.strategies as st
+
+            st.floats(0, 1, False, False, True, 32, True, True)  # allow_subnormal=True
```

Comment:
```suggestion
            st.floats(0, 1, False, False, True, 32, False, False)  # allow_subnormal=True
```

Just to clarify this a bit.